packages/shared/src/lib/featureManagement.ts

```diff
@@ -41,6 +41,8 @@ const feature = {
   }),
 };
 
+export const featurePostTagSorting = new Feature('post_tag_sorting', true);
```

Comment:
Assume you want this :) 
```suggestion
export const featurePostTagSorting = new Feature('post_tag_sorting', false);
```